### PR TITLE
Add RykenSlimeCustomizer

### PR DESCRIPTION
**Pull request by lijinhong11**

### CODE REVIEW

File: pages/Addons.md [hunk 1]
```diff
@@ -95,6 +95,7 @@ GeneticChickengineering-Reborn | kii-chan-reloaded, ybw0014 | <details><summary>
...
 MoreResearches | JustAHuman_xD | <details><summary>expand</summary>MoreResearches is a new simple Slimefun Addon that gives server owners/admins the ability to create custom researches!</details> | [Readme](https://github.com/JustAHuman-xD/MoreResearches#readme) | [Github](https://github.com/JustAHuman-xD/MoreResearches) | [Builds](https://blob.build/project/MoreResearches)
 SlimefunTranslation | ybw0014 | <details><summary>expand</summary>SlimefunTranslation aims to translate Slimefun items (including addons) without modifying the actual item, but the packets.</details> | [Wiki](https://docs.ybw0014.dev/slimefun-translation/) | [GitHub](https://github.com/ybw0014/SlimefunTranslation) | [Builds](https://blob.build/project/SlimefunTranslation)
+RykenSlimeCustomizer | lijinhong11 | <details><summary>expand</summary>RykenSlimeCustomizer can let you customize your Slimefun addon in a new way.</details> | [Wiki](rsc.hiworldmc.com) | [GitHub](https://github.com/SlimefunReloadingProject/RykenSlimeCustomizer-EN) | [Releases](https://github.com/SlimefunReloadingProject/RykenSlimeCustomizer-EN/releases)
```
Comment (Sefiraat) [inline pages/Addons.md:98]:
```suggestion
RykenSlimeCustomizer | lijinhong11 | <details><summary>expand</summary>RykenSlimeCustomizer lets you customize your Slimefun addon in a new way.</details> | [Wiki](rsc.hiworldmc.com) | [GitHub](https://github.com/SlimefunReloadingProject/RykenSlimeCustomizer-EN) | [Releases](https://github.com/SlimefunReloadingProject/RykenSlimeCustomizer-EN/releases)
```

Suggestion, not mandatory - feel free to ignore but likely reads better.

**Comment by Sefiraat** [comment]
The Wiki link resolves to 'File not found' currently. This will need resolving prior to merging.

**Comment by lijinhong11** [comment]
link fixed

**Review by Sefiraat — APPROVED** [review]
LGTM 👍

**Review by svr333 — APPROVED** [review]

**Pull request merged**